cassandra/io/asyncorereactor.py

```diff
@@ -239,6 +267,11 @@ def _run_loop(self):
     def add_timer(self, timer):
         self._timers.add_timer(timer)
 
+        # This function is called from a different thread than the
+        # event loop thread, so for this call is thread safe, we must wake up
```

Comment:
This phrasing confused me at first -- I expected to find a data race or something.

I think I used to understand `wake_loop` enough to rephrase, but I don't anymore :grimacing: